docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,220 @@
+[[scaling-on-kubernetes]]
+= Scaling {agent} on kubernetes
+
+- Please follow the steps to run the {agent} on <<running-on-kubernetes-managed-by-fleet>> page.
+- Please follow the steps to run the {agent} on <<running-on-kubernetes-standalone>> page.
+
+
+
+[discrete]
+== Observability under scale
+
+Using https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-kubernetes.html[Elastic Observability] to monitor Kubernetes infrastructure at scale, imposes a higher complexity in order to configure and adjust Elastic Stack based on different kubernetes sizes. The Horizontal Pod scaling changes the shape of your Kubernetes workload by automatically increasing or decreasing the number of Pods, thus this will impact the following:
+
+- The amount of metrics being collected from several kubernetes endpoints
+- The agent's resources to cope up with the higher CPU and Memory needed for internal processesing
+- The ElasticSearch resources due to higher rate of metric ingestion needs
+- The Dashboard's Visualisations response times as more data are requested on a given time window 
+
+
+[discrete]
+== Configure Agent Resources
+
+The kubernetes observability is based on https://docs.elastic.co/en/integrations/kubernetes[kubernetes integration], which is fetching metrics from several components:
+
+- **Per node:**
+  - kubelet
+  - controller-manager
+  - scheduler
+  - proxy
+- **Cluster Wide (i.e. correspond to the whole cluster):**
+  - kube-state-metrics
+  - apiserver
+
+The Elastic Agent manifest is deployed by default as daemonset. That said, each elastic agent by default is being deployed on every node of kubernetes cluster. 
+
+Additionally by default, one agent is elected as **leader**(more info <<kubernetes_leaderelection-provider>>) in our https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L32[manifests] and this will be responsbile for also collecting the cluster wide metrics
+
+--
+[role="screenshot"]
+image::../images/k8sscaling.png[Agent as daemonset]
+--
+
+Above schema explains how {agent} collect and send metrics to ElasticSearch. Because Leader Agent is responsible to collect also cluster wide metrics this means that requires additional resources.
```

Comment:
```suggestion
The above schema explains how {agent} collects and sends metrics to Elasticsearch. Because of Leader Agent being responsible to also collecting cluster-lever metrics, this means that it requires additional resources.
```